app/controllers/communitarian/verification_intents_controller.rb

```diff
@@ -29,11 +29,11 @@ def show
     private
 
     def verification_intent
-      @verification_intent ||= Communitarian::Stripe.new.get_verification_intent(params[:id])
```

Comment:
we need memoization here because we using this method 2 times in `def show` :
- `billing_address `
- `render_verification_intent_result(verification_intent)`